### ISSUE TITLE
"Cannot find module './lib/big' from Decimal.js" with fable-splitter

### ISSUE THREAD

**Issue by xdaDaveShaw** [fable-compiler/Fable#1700]
### Description

I've just updated one of my repos to Fable 2.1.9 from 2.1.6 and I am no long able to run my Jest tests against the output of fable-splitter.

I get the following error:

```
Cannot find module './lib/big' from 'Decimal.js'

      27 | exports.get_MinValue = exports.get_MaxValue = exports.get_MinusOne = exports.get_One = exports.get_Zero = exports.default = void 0;
      28 | 
    > 29 | var _big = _interopRequireDefault(require("./lib/big"));
         |                                   ^
      30 | 
      31 | function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
      32 | 

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:221:17)
      at Object.require (tests/output/fable-library.2.1.9/Decimal.js:29:35)

```
### Repro code

Look at the my XmasList repro on the [fable-2-1-9 branch](https://github.com/xdaDaveShaw/XmasList/tree/fable-2-1-9)
Just running `./build.sh` gives the error.

### Related information

* fable-compiler: 2.1.9
* fable-loader: 2.1.2
* fable-splitter: 2.1.1
* Operating system: Ubuntu 18.04

**Comment by alfonsogarciacaro** [comment]
Thanks for the report @xdaDaveShaw! This looks like a regression in fable-splitter 2.1.1, if you install 2.1.0 instead it will work.

I'll try to fix the problem or revert the publish, cc @nojaf

**Comment by xdaDaveShaw** [comment]
Fixed in fable-splitter 2.1.2

Thanks
